composer.json

```diff
@@ -35,6 +35,9 @@
       "vendor/plugins/{$name}/": ["type:wordpress-plugin"]
     }
   },
+  "require": {
+    "inwerpsel/github-archive-installer": "^1.2.1"
```

Comment:
Can we bring this to `greenpeace` to give access to the whole team? in case more changes needed in the future.

Reply:
Of course, I just made the fork on my account because I wasn't sure if it would work :+1: